streams/src/main/java/org/apache/kafka/streams/kstream/internals/Change.java

```diff
@@ -30,7 +30,7 @@ public Change(final T newValue, final T oldValue) {
 
     @Override
     public String toString() {
-        return "(" + String.valueOf(newValue) + "<-" + String.valueOf(oldValue) + ")";
+        return "(" + newValue + "<-" + oldValue + ")";
```

Comment:
@mjsax pointed out that, as of Java 8, this "safety measure" is unnecessary, since Java now automatically converts `null` to `"null"` in string concatenation.

reverts: https://github.com/apache/kafka/pull/6848/files#r289229094